src/modules/peek/Peek.FilePreviewer/Previewers/PreviewerFactory.cs

```diff
@@ -10,24 +10,13 @@ public class PreviewerFactory
     {
         public IPreviewer? Create(File file)
         {
-            // TODO: investigate performance of reflection to resolve previewer type
-            switch (file.Extension)
+            if (ImagePreviewer.IsFileTypeSupported(file.Extension.ToLower()))
```

Comment:
1. Consider moving ToLower to the class `File` so we don't have to do it everywhere.